Tests/fixtures/tv-3.yml

```diff
@@ -2079,3 +2079,513 @@
+  browser_family: Android Browser
+-
+  user_agent: Kodi/19.3 (Linux; Android 9.0; MAG425A Build/PPR1.180610.011) Android/9.0.0 Sys_CPU/armv7l App_Bitness/32 Version/19.3-(19.3.0)
+  os:
+    name: Android
+    version: "9.0"
+    platform: ARM
+  client:
+    type: mediaplayer
+    name: Kodi
+    version: "19.3"
+  device:
+    type: tv
+    brand: Infomir
+    model: MAG425A
+  os_family: Android
+  browser_family: Unknown
+-
+  user_agent: TiviMate/4.7.0 (SDMC D1221; Android 11)
+  os:
+    name: Android
+    version: "11"
+    platform: ""
+  client:
+    type: mobile app
+    name: TiviMate
+    version: 4.7.0
+  device:
+    type: tv
+    brand: ETOE
+    model: D1221
+  os_family: Android
+  browser_family: Unknown
+-
+  user_agent: Dalvik/2.1.0 (Linux; U; Android 9; IMAQ TETRA Build/PPR1.180610.011)
+  os:
+    name: Android
+    version: "9"
+    platform: ""
+  client:
+    type: browser
+    name: Android Browser
+    version: ""
+    engine: WebKit
+    engine_version: ""
+  device:
+    type: tv
+    brand: Imaq
+    model: Tetra
+  os_family: Android
+  browser_family: Android Browser
+-
+  user_agent: Mozilla/5.0 (Linux; Android 7.1.2; Amlogic Alvatech GT1 Build/NHG47L) IPTV Pro/7.1.3
+  os:
+    name: Android
+    version: 7.1.2
+    platform: ""
+  client:
+    type: mobile app
+    name: IPTV Pro
+    version: 7.1.3
+  device:
+    type: tv
+    brand: Beelink
+    model: GT1
+  os_family: Android
+  browser_family: Unknown
+-
+  user_agent: Dalvik/2.1.0 (Linux; U; Android 7.1.2; COOD-E TV 4K Build/NHG47L)
+  os:
+    name: Android
+    version: 7.1.2
+    platform: ""
+  client:
+    type: browser
+    name: Android Browser
+    version: ""
+    engine: WebKit
+    engine_version: ""
+  device:
+    type: tv
+    brand: COOD-E
+    model: TV Box 4K
+  os_family: Android
+  browser_family: Android Browser
+-
+  user_agent: Dalvik/2.1.0 (Linux; U; Android 9; D2140 Build/PI)
+  os:
+    name: Android
+    version: "9"
+    platform: ""
+  client:
+    type: browser
+    name: Android Browser
+    version: ""
+    engine: WebKit
+    engine_version: ""
+  device:
+    type: tv
+    brand: Anker
+    model: Nebula Cosmos
+  os_family: Android
+  browser_family: Android Browser
+-
+  user_agent: Dalvik/2.1.0 (Linux; U; Android 9; D2150 Build/PI)
+  os:
+    name: Android
+    version: "9"
+    platform: ""
+  client:
+    type: browser
+    name: Android Browser
+    version: ""
+    engine: WebKit
+    engine_version: ""
+  device:
+    type: tv
+    brand: Anker
+    model: Nebula Cosmos Max
+  os_family: Android
+  browser_family: Android Browser
+-
+  user_agent: Dalvik/2.1.0 (Linux; U; Android 11; AI PONT Build/RTO2.220913.001)
+  os:
+    name: Android
+    version: "11"
+    platform: ""
+  client:
+    type: browser
+    name: Android Browser
+    version: ""
+    engine: WebKit
+    engine_version: ""
+  device:
+    type: tv
+    brand: Changhong
+    model: AI PONT
+  os_family: Android
+  browser_family: Android Browser
```

Comment:
https://storage.googleapis.com/play_public/supported_devices.html

I would exclude this from a specific brand and move it to Unknown.
Since it is impossible to determine exactly what this DRM device is